### PR TITLE
Fix magnet URIs handling in client.py

### PR DESCRIPTION
**Pull request by evsh**

### CODE REVIEW

File: bindings/python/client.py [hunk 1]
```diff
@@ -156,7 +156,8 @@ def print_download_queue(console, download_queue):
 def add_torrent(ses, filename, options):
     atp = lt.add_torrent_params()
     if filename.startswith('magnet:'):
-        atp = lt.parse_magnet_uti(filename)
+        for key, value in lt.parse_magnet_uri(filename).items():
+            setattr(atp, key, type(getattr(atp, key))(value))
```
Comment (arvidn) [inline bindings/python/client.py]: did the return value from `parse_magnet_uri()` not work when passing into `add_torrent()`?

Comment (evsh) [reply bindings/python/client.py]: No, it did not.

Comment (arvidn) [reply bindings/python/client.py]: it's supposed to work. both `add_torrent()` and `async_add_torrent()` are overloaded to support both a dict and the `add_torrent_params` object. what's the error you got?

Comment (evsh) [reply bindings/python/client.py]: The problem is not in the `add_torrent()`. The code below the changed fragment expects `atp` to hold an `add_torrent_params` but not `dict` object:
```
Traceback (most recent call last):
  File "client.py", line 360, in <module>
    main()
  File "client.py", line 237, in main
    add_torrent(ses, f, options)
  File "client.py", line 167, in add_torrent
    atp.save_path = options.save_path
AttributeError: 'dict' object has no attribute 'save_path'
```

Comment (evsh) [reply bindings/python/client.py]: P.S. This is after fixing the typo in `parse_magnet_uTi`

Comment (arvidn) [reply bindings/python/client.py]: I see, `parse_magnet_uri()` returns a dict. I suppose this is historical. I will make it return `add_torrent_params`

**Comment by arvidn** [comment]
I fixed this in RC_1_1 and merged it into master [here](https://github.com/arvidn/libtorrent/commit/fc56ec194ae1e189cf7ae9f22d857a807e49a439). I think this supersedes this patch

**Comment by evsh** [comment]
yep. thanks!

**Pull request closed without merging**